starbucks/src/main/java/git_kkalnane/backend/starbucks/store/domain/Store.java

```diff
@@ -0,0 +1,56 @@
+package git_kkalnane.backend.starbucks.store.domain;
+
+import git_kkalnane.backend.starbucks.global.entity.BaseTimeEntity;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "stores")
+@Getter
+@Setter
```

Comment:
특정 멤버 필드에 대한 Setter는 필요시 정의하되, Lombok Setter는 지양해주세요!! 
그리고, Builder 어노테이션을 사용하여, 엔티티 구성하실 때 좀 더 편하게 작성하실 수 있을겁니다. 다른 방식도 있지만 간단하게 소개드릴 수 있는건 Builder 패턴이네요.

다음과 같이 제안드립니다.

```suggestion
@Builder
```

Reply:
네, 피드백 감사합니다! 엔티티의 불변성을 높이고 객체 생성을 더 명확하게 하기 위해 @Setter를 지양하고 @Builder를 활용하도록 수정하겠습니다.